content/en/01-about-pixie/04-faq.md

```diff
@@ -43,47 +45,73 @@ order: 4
 
-Pixie stores the data it collects on the cluster in-memory; data is not sent to any centralized backend outside of the cluster.
+Pixie stores the data it collects in-memory on the nodes in your cluster; no data is sent to a centralized backend outside of the cluster. This is true for both self-hosted Pixie and [Pixie Community Cloud](/about-pixie/faq#does-pixie-offer-a-hosted-cloud-offering).
+
+Pixie has a [2GiB memory requirement](/installing-pixie/requirements/#memory) per node. After installing Pixie, it is normal to see a temporary increase in memory usage of the `vizier-pem` pods as they begin to fill their data tables.
+
+### How much data does Pixie store?
 
 Retention time depends on the level of traffic in your cluster, but will generally be on the order of hours. Each data table in Pixie (e.g. `http_events`) has its own maximum size. These tables collect data until the maximum size is reached, and then begin dropping the oldest data.
 
-We recommend integrating with third-party observability tools (such as New Relic’s integration) to provide long-term retention.
+We recommend integrating with third-party observability tools (such as [New Relic’s integration](https://newrelic.com/platform/kubernetes-pixie)) to provide long-term retention.
+
+### How does Pixie secure its data?
+
+Pixie stores the telemetry data it collects in-memory on the nodes in your cluster. Data processing and script execution are also performed in the cluster. This is true for both self-hosted Pixie and [Pixie Community Cloud](/about-pixie/faq#does-pixie-offer-a-hosted-cloud-offering).
+
+End-to-end encryption is offered for data in flight between in-cluster storage and presentation in the UI / CLI / API:
+
+- **Live UI** (enabled by default)
+- **CLI** (enabled by default, [option to disable](/using-pixie/using-cli/#disabling-encryption) for faster script execution)
+- **API** (disabled by default, [enable during client creation](/using-pixie/api-quick-start/#encryption))
+
+Pixie Cloud (self-hosted or Pixie Community Cloud) hosts the UI and stores limited metadata related to account (user, organization) and Kubernetes control data (cluster name, number of nodes, etc). All communication with Pixie Cloud is TLS encrypted.
+
+Pixie supports two modes for accessing data by the UI. In Passthrough Mode, data flows through Pixie's Cloud via a reverse proxy as encrypted traffic without any persistence. This convenience mode allows developers to access data without being in the same network as the cluster. In Data Isolation Mode, no data flows through Pixie Cloud, instead the browser directly proxies into Pixie's Vizier Module. For more info, see [Configuring Data Transfer Modes](/reference/admin/data-transfer-mode/).
+
+_Community Cloud for Pixie Security_
+
+Pixie Community Cloud is a hosted version of Pixie Cloud. Customer metadata will only be accessed by a limited production team for support purposes. All access is logged in an internal tool.
```

Comment:
Maybe also reiterate that Pixie team members will not have any access to customer data. 

(We can use our support accounts only if they give us permission, but I don't know if we need to include that here).